src/components/Table/table-cells/TableBodyCell.tsx

```diff
@@ -15,16 +14,67 @@ export const TableBodyCell: React.FC<TableCellProps> = ({
     className,
     iconStart,
     iconEnd,
-    isFirstColumn,
-    canExpand,
-    isExpanded,
-    depth,
-    getToggleRowExpandedProps,
+    expandableColumnIndex,
+    row,
+    cell,
     children,
     ...props
 }) => {
-    const paddingLeft = `${depth * 0.75}rem`;
-    const marginLeft = spacings.spacingMedium;
+    // Fix this type definition of any
+    const isExpandableColumn = (row as any)?.cells[expandableColumnIndex].value === cell?.value;
+    const leftMarginBuffer = 20; // px
+    const leftMarginIndentDepth = row?.depth * 20; // px
```

Comment:
Do you think is it relevant to use `leftMarginBuffer` in place of the numeric value `20`.
```suggestion
    const leftMarginIndentDepth = row?.depth * leftMarginBuffer; // px
```

Reply:
The only reason I thought so was to have all the constant values easily located at the top of the component. But, I'm happy to go either way on this.